master/docs/manual/cfg-reporters.rst

```diff
@@ -253,6 +253,10 @@ MailNotifier arguments
     When this argument is ``True`` (default is ``False``) ``MailNotifier`` sends emails using TLS and authenticates with the ``relayhost``.
     When using TLS the arguments ``smtpUser`` and ``smtpPassword`` must also be specified.
 
+``useSsl``
+    (boolean).
+    When this argument is ``True`` (default is ``False``) ``MailNotifier`` sends emails using SSL and authenticates with the ``relayhost``.
```

Comment:
I think the `useSsl` parameter should be renamed. Perhaps `connectSsl` to match that `connectSSL` is used or `connectTls` because SSL is completely deprecated now.

The `useTls` option controls `requireTransportSecurity` which makes Twisted require that STARTTLS is used or that the connection is over SSL/TLS in the first place. That option in MailNotifier should also be renamed but too late.